test/unit/render/glyph_manager.test.js

```diff
@@ -21,7 +31,7 @@ const createLoadGlyphRangeStub = (t) => {
 };
 
 const createGlyphManager = (font) => {
-    const manager = new GlyphManager(identityTransform, font);
+    const manager = new GlyphManager(identityTransform, font ? LocalGlyphMode.ideographs : LocalGlyphMode.none, font);
     manager.setURL('https://localhost/fonts/v1/{fontstack}/{range}.pbf');
```

Comment:
Are we gonna add tests for LocalGlyphMode.all in the future?

Reply:
Good call, I added a basic test.